testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -573,18 +573,20 @@ def test_sklearn_affinity_propagation(self, ens1):
                               "clusters: {0} {1}".format(len(cc1), len(cc2))
 
     def test_ClusterCollection_init(self, cc):
-        assert np.all(cc.clusters[0].elements == [0, 1, 2]) and \
-               np.all(cc.clusters[1].elements == [3, 4   ]) and \
-               np.all(cc.clusters[2].elements == [5, 6, 7]) and \
+        if np.testing.assert_allclose(cc.clusters[0].elements , [0,1,2])and \
+               np.testing.assert_allclose(cc.clusters[1].elements , [3, 4]) \
+                   and \
+               np.testing.assert_allclose(cc.clusters[2].elements , [5, 6, 7])\
+                   and \
```

Comment:
`assert_allclose` will already 'fail" when one condition is not full filled. No need of an `if` statement

```suggestion
        assert_allclose(cc.clusters[0].elements , [0,1,2])
        assert_allclose(cc.clusters[1].elements , [3, 4])
        assert_allclose(cc.clusters[2].elements , [5, 6, 7])
```

I would also split the `cc.clusters[0].centroid == 1` into an assert like assert `cc.clusters[0].centroid == 1`. This makes the whole structure easier to read and the print statement superflous.